src/App/Enumeration.cpp

```diff
@@ -149,11 +149,11 @@ void Enumeration::setEnums(const std::vector<std::string> &values)
 
     _EnumArray = new const char*[values.size() + 1];
     int i = 0;
-    for (std::vector<std::string>::const_iterator it = values.begin(); it != values.end(); ++it) {
+    for (const auto & value : values) {
 #if defined (_MSC_VER)
         _EnumArray[i++] = _strdup(it->c_str());
```

Comment:
You need to update the other half of this #if

Reply:
Grrr. Thanks. IDE is hiding stuff.